activestorage/lib/active_storage/migration.rb

```diff
@@ -0,0 +1,27 @@
+class ActiveStorageCreateTables < ActiveRecord::Migration[5.1]
```

Comment:
Do you think this class should be `nodoc` or needs to be documented?
Currently it's showing up when running `rake rdoc` at the very root (since it's not namespaced):

<img width="313" alt="list" src="https://user-images.githubusercontent.com/10076/28939973-eab68490-7847-11e7-9566-221874b22edf.png">